sql/core/src/main/scala/org/apache/spark/sql/execution/CacheManager.scala

```diff
@@ -181,6 +182,38 @@ class CacheManager extends Logging {
     }
   }
 
+  private[sql] def analyzeColumnCacheQuery(
+      query: Dataset[_],
+      columnNames: Seq[Attribute]): Unit = writeLock {
```

Comment:
`columnNames` -> `columns` because it becomes `Seq[Attribute]` instead of `Seq[String]`.